charts/stable/unpoller/Chart.yaml

```diff
@@ -21,8 +21,9 @@ name: unpoller
 sources:
   - https://github.com/truecharts/charts/tree/master/charts/stable/unpoller
   - https://github.com/unifi-poller/unifi-poller
+  - https://ghcr.io/unpoller/unpoller
 type: application
-version: 5.0.8
+version: 6.0.0
```

Comment:
why is this a major change?

Reply:
because it was changed to a different image container

Reply:
Lets not do that and keep it it a minor version change like the container.